queries/http/highlights.scm

```diff
@@ -24,17 +35,19 @@
   "?"
   "&"
   "@"
+  "<"
 ] @operator
 
 ; Literals
-(string) @string
-
 (target_url) @string.special.url
 
+(http_version) @constant
+
+(string) @string
+
 (number) @number
 
-; (boolean) @boolean
-(null) @constant.builtin
```

Comment:
So what happened to null?

Reply:
That node had been a cleanup error when I made the initial parser version to be included here, since before the parser had its own JSON parsing. After that, the node was removed and ceased to exist in the grammar since it had no use case, so I've removed it from the highlights query as well.